src/frontend/komponenter/Fagsak/Vedtak/FeilutbetaltValuta/FeilutbetaltValutaPeriode.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Delete } from '@navikt/ds-icons';
+import { Table, Button, Tooltip, Alert } from '@navikt/ds-react';
+
+import type { IRestFeilutbetaltValuta } from '../../../../typer/eøs-feilutbetalt-valuta';
+import { periodeToString } from '../../../../utils/kalender';
+import FeilutbetaltValutaSkjema from './FeilutbetaltValutaSkjema';
+import { useFeilutbetaltValuta } from './useFeilutbetaltValuta';
+
+interface IFeilutbetaltValutaPeriode {
+    feilutbetaltValuta: IRestFeilutbetaltValuta;
+    erLesevisning: boolean;
+    behandlingId: number;
+}
+
+const FlexColumnDiv = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    margin-bottom: 2rem;
+`;
+
+const FlexRowDiv = styled.div`
+    display: flex;
+    gap: 1rem;
+`;
+
+const FeilutbetaltValutaPeriode: React.FC<IFeilutbetaltValutaPeriode> = ({
+    feilutbetaltValuta,
+    erLesevisning,
+    behandlingId,
+}) => {
+    const [erRadEkspandert, settErRadEkspandert] = useState<boolean>(false);
+    const [feilmelding, settFeilmelding] = useState<string>();
+
+    const { skjema, oppdaterEksisterendePeriode, nullstillSkjema, fjernPeriode, valideringErOk } =
+        useFeilutbetaltValuta({
+            behandlingId: behandlingId,
+            feilutbetaltValuta,
+            settFeilmelding: settFeilmelding,
+        });
+
+    useEffect(() => {
+        nullstillOgLukkSkjema();
+    }, [feilutbetaltValuta]);
```

Comment:
Tror React-folka ønsker at man unngår å bruke useEffect når en prop endres: https://beta.reactjs.org/learn/you-might-not-need-an-effect#chains-of-computations